src/test/java/de/rabea/response/creator/OptionsResponseCreatorTest.java

```diff
@@ -0,0 +1,32 @@
+package de.rabea.response.creator;
+
+import de.rabea.request.HttpVerb;
+import de.rabea.response.ResponseCreator;
+import de.rabea.response.head.FakeResponseCreator;
+import de.rabea.response.head.OptionsResponseHeader;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static de.rabea.request.HttpVerb.GET;
+import static de.rabea.request.HttpVerb.POST;
+import static de.rabea.response.head.StatusLine.OK;
+import static org.junit.Assert.assertTrue;
+
+public class OptionsResponseCreatorTest {
+
+    @Test
+    public void createResponse() {
+        Map<HttpVerb, ResponseCreator> responses = new HashMap<>();
+        responses.put(GET, new FakeResponseCreator());
+        responses.put(POST, new FakeResponseCreator());
+        OptionsResponseCreator creator = new OptionsResponseCreator(OK, new OptionsResponseHeader(responses));
```

Comment:
OptionsResponseHeader is only used in combination with the OptionsResponseCreator... you may be able to meld those classes together